website/templates/project/settings.mako

```diff
@@ -387,6 +411,8 @@
       window.contextVars.node.description = ${node['description'] | sjson, n };
       window.contextVars.node.nodeType = ${ node['node_type'] | sjson, n };
       window.contextVars.nodeCategories = ${ categories | sjson, n };
+      window.contextVars.wiki = window.contextVars.wiki || {};
```

Comment:
If I add that, I get the following when I load the page.

Error !

TypeError: <website.addons.base.AddonConfig object at 0x1141a83d0> is not JSON serializable